docs/howtouse/sitename.rst

```diff
@@ -1,35 +1,22 @@
-Adding Sites
-============
+Updating the Domain of Your Site
+================================
 
 If you are running the record transfer app somewhere other than your localhost, you will need to add
 your site's domain to the database. If you do not set the domain, your emails will not send
 correctly, and will have the wrong link to your site.
 
-To set the name and domain of your site, first log in as a **superuser**. Regular staff users do not
-have permission to modify Sites.
+To set the domain of your site, run the following command from the `app` directory:
 
-Go to the administrator application (accessible at /admin/), and click on **+ Add** next to **Sites**.
+.. code-block:: shell
 
-.. image:: images/admin_add_site.png
-    :alt: Green circle around Add Sites link in admin app
+    python manage.py set_domain "https://my.domain.com"
 
-Fill out your domain name, and give the website a name (you can change the name later if you don't
-like it). Once filled out, click the blue **Save and continue** button.
+replacing `https://my.domain.com` with your site's domain.
```

Comment:
I don't love how restructured text has you specify code inline, but you have to do it like this:

```suggestion
replacing code:`my.domain.com` with your site's fully qualified domain name.
```